Source/SteveSloth/Public/MyEnemyAIIdleState.h

```diff
@@ -0,0 +1,36 @@
+/****************************************************************************************
+* Copyright: SteveSloth
+ * Name: Aidan MacLeod
+ * Script: MyEnemyAIIdleState.h
+ * Date: April 29. 2024
+ * Description: Idle State Class for all enemies to inherit from
+ * TODO:
+ * Known Bugs:
+ ****************************************************************************************/
+
+#pragma once
+
+#include "CoreMinimal.h"
+
+/**
+ * 
+ */
+
+UCLASS()
```

Comment:
For the entirety of the header file.. please follow the template that was put in chat.. including comments for sections